geomesa-index-api/src/main/scala/org/locationtech/geomesa/index/conf/QueryHints.scala

```diff
@@ -60,6 +60,7 @@ object QueryHints {
   val ARROW_FORMAT_VERSION     = new ClassKey(classOf[String])
   val ARROW_DICTIONARY_FIELDS  = new ClassKey(classOf[java.lang.String])
   val ARROW_PROCESS_DELTAS     = new ClassKey(classOf[java.lang.Boolean])
+  val ARROW_FLATTEN            = new ClassKey(classOf[java.lang.Boolean])
```

Comment:
could we rename this maybe `ARROW_FLATTEN_COLS`? not sure about COLS as a qualifier, exactly, but seems like it should be more descriptive on what is being flattened.

Reply:
The GS part of this calls it `FLATTENSTRUCT ` https://github.com/geomesa/geomesa-geoserver/pull/55/files#diff-d60681225c5f7798d96ec344af655904f41b6d13260fa0bcb7a2a3f0048ceb14R162

Do you like `ARROW_FLATTEN _COLS` or `ARROW_FLATTEN_STRUCT` here? I can update the GS MR to match whichever way.

Reply:
Great, let me know if that looks better. I addressed this and your previous comments.